content/documentation/pages/1-installation/3-kubernetes/2-kubectl.md

```diff
@@ -158,32 +165,32 @@ grafana-info:
 
 <!--END_TIP-->
 
-The default Grafana dashboard credentials are a username of `admin` and
-a password of `password`. You can change these defaults by modifying the
-`src/kubernetes/grafana/grafana-secret.yaml` file.
+The default Grafana dashboard credentials are a username of `admin` and a password of `password`. You can change these defaults by modifying the `src/kubernetes/grafana/grafana-secret.yaml` file.
 
-In the event that you would not like to deploy metrics collection by
-using Prometheus and Grafana, you should remove the following section of
-`src/kubernetes/server/server-config.yaml`. You can edit the
-appropriate file based on the messaging middleware deployed earlier:
+In the event that you would not like to deploy metrics collection by using Prometheus and Grafana, you should remove the following section of `src/kubernetes/server/server-config.yaml`:
 
-```yml
+```yaml
 applicationProperties:
   stream:
     management:
       metrics:
         export:
           prometheus:
             enabled: true
-      endpoints:
```

Comment:
If we are advising to remove the exposure of endpoints, why even add the `prometheus-proxy` section below?